src/test/java/com/google/devtools/build/lib/remote/RemoteExecutionServiceTest.java

```diff
@@ -1834,30 +1834,6 @@ public void buildRemoteActionForRemotePersistentWorkers() throws Exception {
                         .setValue(
                             "b22d48cd55755474eae27e63a79306a64146bd5947d5bd3423d78f001cf7b3de"))
                 .build());
-    var merkleTree = remoteAction.getMerkleTree();
-    var outputDirectory =
-        merkleTree.getDirectoryByDigest(merkleTree.getRootProto().getDirectories(0).getDigest());
-    var inputFile =
-        FileNode.newBuilder()
-            .setName("input")
-            .setDigest(
-                Digest.newBuilder()
-                    .setHash("cd42404d52ad55ccfa9aca4adc828aa5800ad9d385a0671fbcbf724118320619")
-                    .setSizeBytes(5))
-            .setIsExecutable(true);
-    var toolFile =
-        FileNode.newBuilder()
-            .setName("worker_input")
-            .setDigest(
-                Digest.newBuilder()
-                    .setHash("bbd21d9e9b2bbadb2bb67202833df0edc8d14baf38be49388ffc71831eb88ac4")
-                    .setSizeBytes(12))
-            .setIsExecutable(true)
-            .setNodeProperties(
-                NodeProperties.newBuilder()
-                    .addProperties(NodeProperty.newBuilder().setName("bazel_tool_input")));
-    assertThat(outputDirectory)
```

Comment:
Please add a new test case for this assert.

Reply:
With the flag added, I have been able to revert this test entirely.